phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexTool.java

```diff
@@ -147,7 +146,7 @@
             "To build indexes for a data table from least disabledTimeStamp");
     
     private static final Option DIRECT_API_OPTION = new Option("direct", "direct", false,
-            "If specified, we avoid the bulk load (optional)");
+            "This option is no longer a parameter. Direct is mandatory. Keeping as parameter due to backwards compatibility (optional)");
```

Comment:
How about "This parameter is deprecated. Direct mode will be used whether it is set or not. Keeping it for backwards compatibility."